docs/src/views/FAQ.tsx

```diff
@@ -40,6 +69,20 @@ export default () => (
       }
     />
 
+    <Question content="How can I set default value of Form.Field?" />
+    <Answer
+      content={
+        <p>
+          You need to set the default value on the input slot.
```

Comment:
I would rephrase this to:
`You need to set the default value on the component used in the control slot.`
The reason for this is that, in the control slot we may add input, radio, checkbox components that may implement this props in a different way.